### DISCUSSION TITLE
Download folder location

### DISCUSSION THREAD

**Discussion by macuser2** [UltimaHoarder/UltimaScraper#1033] [Q&A]
So as it is right now, all the content is downloaded to ".sites" & then is sorted individually into model folders. 

Is there anyway I can change the download location to another folder on my hard drive or rename the folder to just "sites" or would that require the entire script to be changed?

**Comment by headphonia** [comment]
You can change the download folder using the `download_directories:`parametre in the config file.

**Reply by macuser2** [reply]
So if I change the download directory to just the folder name it won't mess with the actual code for scraping?

**Reply by headphonia** [reply]
No, it shouldn't mess with the actual code. Just remember to rename the actual folder before scraping again, otherwise it will redownload everything.
